src/components/map.js

```diff
@@ -88,6 +88,8 @@ class map extends Component {
 							<option value="UI/UX">UI/UX</option>
 							<option value="Web Dev">Web Dev</option>
 							<option value="IoT">IoT</option>
+							<option value="XR">IoT</option>
```

Comment:
Here IoT needs to be replaced with XR

Reply:
@abhinavsri360 Also here, don't you feel it should be `AR/VR/XR` instead of just `XR`? There are people who don't know all 3 terms. Most are familiar with only `AR` or `VR`. And all 3 come in the same category with `XR` more as a superset of the two.